examples/sample_policy.yaml

```diff
@@ -1,27 +1,36 @@
 nodes:
   listener:
     topics:
-      chatter:
-        allow: s # can subscribe to chatter
+      /chatter:
```

Comment:
Unless it has become a requirement that topic names should start with the global namespace, `'/'`, I don't think this change should be necessary.

Reply:
Private topic names will be specified from global with the entire path to avoid confusion in the current revision. Is this something that is not desired?

Reply:
It can either have the / or not, but they will be globally referenced